app/models/policies/international_relocation_payments/eligibility.rb

```diff
@@ -3,8 +3,21 @@ module InternationalRelocationPayments
     class Eligibility < ApplicationRecord
       self.table_name = "international_relocation_payments_eligibilities"
 
+      AMENDABLE_ATTRIBUTES = %i[].freeze
+
       has_one :claim, as: :eligibility, inverse_of: :eligibility
 
+      attr_accessor :teacher_reference_number
+
+      def award_amount
+        0
+      end
+
+      # No current_school attribute on the model. This method is for compatibility with the admin UI.
+      def current_school
+        nil
+      end
```

Comment:
Not a blocker but we can now change this to

```ruby
belong_to :current_school, class_name: "School"
```

as the current_school_id is being set on master